v2/component/launcher.go

```diff
@@ -727,3 +737,17 @@ func (l *Launcher) openBucket() (*blob.Bucket, error) {
 	}
 	return blob.OpenBucket(context.Background(), l.bucketConfig.bucketURL())
 }
+
+func getMinioCredential() (*v1.Secret, error) {
+	restConfig, err := rest.InClusterConfig()
+	if err != nil {
+		return nil, fmt.Errorf("Failed to initialize kubernetes client: %v", err)
+	}
+	clientSet, err := kubernetes.NewForConfig(restConfig)
+	if err != nil {
+		return nil, fmt.Errorf("Failed to initialize kubernetes client set: %v", err)
+	}
+	namespace := os.Getenv("KFP_NAMESPACE")
```

Comment:
nit return error when env not exist or is empty?

Reply:
done